### PR TITLE
Added Light Module and Attribute Support to the Chamberlain MyQ Binding

### PR DESCRIPTION
**Pull request by computergeek1507**
See Forum for more details
https://community.openhab.org/t/chamberlain-myq-binding/4192/68

### CODE REVIEW

File: bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/LampDevice.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+							"Lamp DeviceID: {} DeviceType: {} Lightstate : {}",
+							deviceId, deviceType, lightstate);
+					break;
+				}
+			}
+		}
+	}
+
+	/**
+	 * Internal Method to convert API valie to ON/OFF state
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/LampDevice.java]: "value"

File: bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java [hunk 2]
```diff
@@ -39,200 +39,229 @@
...
+	}
+
+	/**
+	 * Retrieves MyQ device data from myq website, throws if connection
+	 * fails or user login fails
+	 * 
+	 */
+	public MyqDeviceData getMyqData() throws InvalidLoginException,
+			IOException {
+		logger.trace("Retreiveing door data");
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java]: "Retrieving"

File: bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqDeviceData.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+/**
+ * This Class parses the JSON data and creates a LinkedList of MyQ 
+ * Devices
+ * <ul>
+ * <li>devices: LinkedList of Devices</li>
+ * </ul>
+ * 
+ * @author Scott Hanson
+ * @since 1.8.0
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqDeviceData.java]: @since 1.9.0

**Comment by computergeek1507** [comment]
@9037568 I tried to fix all your comments and added some other code I was testing.

**Comment by 9037568** [comment]
Thanks, @computergeek1507 !

This was a bit of a complex merge. Can you please verify the version committed is a good match for your intended changes?

**Comment by computergeek1507** [comment]
@9037568 I'm not sure what you mean but the latest commit has the new added features and a couple bug fixes other users have reported in the forums. I am currently running it on my main install (1.8.3) and tested it on the newest 2.0.0 snapshot without any issues.

**Comment by 9037568** [comment]
I had to resolve a fair number of merge conflicts when merging this PR. I was just wanting to verify the code that now exists in the master branch matches what you intended.

I've now verified that the hard way, so you can ignore me.

**Pull request closed without merging**